cryptosheets.py

```diff
@@ -71,6 +71,8 @@ def main():
                          'price_btc',
                          'price_usd',
                          'percent_total_market']]
+    write_df = write_df.sort_values(by='percent_total_market')
+    print(f'the cryptocurrency with the biggest market cap today is: {write_df['name'][0]}')
```

Comment:
assign the name of the highest market cap cryptocurrency to a variable before printing. I think this would be more readable.